OpenOversight/app/util/general.py

```diff
@@ -0,0 +1,164 @@
+import random
+import sys
+from distutils.util import strtobool
+from typing import Optional
+from urllib.parse import urlparse
+
+from flask import current_app, url_for
+from future.utils import iteritems
+
+from ...app.custom import add_jpeg_patch
+
+
+# Call JPEG patch function
+add_jpeg_patch()
+
+
+def ac_can_edit_officer(officer, ac):
+    if officer.department_id == ac.ac_department_id:
+        return True
+    return False
+
+
+def allowed_file(filename):
+    return (
+        "." in filename
+        and filename.rsplit(".", 1)[1].lower()
+        in current_app.config["ALLOWED_EXTENSIONS"]
+    )
+
+
+def get_or_create(session, model, defaults=None, **kwargs):
```

Comment:
If we're good now, I think we can keep it where it's at. I'll add a note to the top of it tho signaling that it could be something we move to its own file should a circular import arise.